src/Temporalio/Runtime/TemporalRuntime.cs

```diff
@@ -13,6 +13,11 @@ namespace Temporalio.Runtime
     /// </remarks>
     public sealed class TemporalRuntime
     {
+        /// <summary>
+        /// Prefix for reserved handler and definition names.
+        /// </summary>
+        internal const string ReservedNamePrefix = "__temporal";
```

Comment:
Shouldn't it be `__temporal_` per https://github.com/temporalio/features/issues/576

Reply:
See up here: https://github.com/temporalio/sdk-dotnet/pull/410#discussion_r1949907394

Yeah, that's what's written there, but, also that just seems wrong. Probably we should change that spec.